apps/stripe/src/app/api/webhooks/stripe/route.ts

```diff
@@ -82,26 +81,27 @@ const StripeWebhookHandler = async (request: NextRequest): Promise<Response> =>
 
     /**
      * todo:
-     * - attach operations context to Response, so we can print them to logger here
+     * - attach operations context to Response, so we can print them to logger here (like what was reported)
      */
     return result.match(
       (success) => {
-        logger.info("Success processing Stripe webhook");
+        logger.info("Success processing Stripe webhook", {
+          httpsStatusCode: success.responseStatusCode,
```

Comment:
Suggestion: can we add amount / reported result (e.g `CHARGE_SUCCESS`) as well?

Reply:
We dont pass it to route yet but I added extra log 

```
    this.logger.info("Transaction event reported", {
      transactionId: processingResult.value.saleorTransactionId,
      amount: processingResult.value.saleorMoney.amount,
      result: processingResult.value.transactionResult,
    });```